blog/BLOG.md

```diff
@@ -50,24 +50,31 @@ Cerberus is a collection of the following tasks that work together to achieve a
 
 ## Where to go from here?
 
-Cerberus is compatible with any project that uses [fastlane] or [gradle] and [Jira] as the project tracker. If you would like to give it a spin:
+Cerberus is compatible with any project that uses [Fastlane] or [Gradle] and [Jira] as the project tracker. Here are some links if you would like to give it a spin:
 
-! Add pending links and instructions here!
+**Fastlane**
 
+- [Project Page](https://github.com/outware/fastlane-plugin-cerberus)
```

Comment:
Was there a reason why you decided to keep these urls adjacent to links rather that separating them as you have done with other links?

Was it because they are not displayed inline with in a sentence or something else?

Reply:
I separate URLs when it used in more than one place. I can use `[Jira]` anywhere and it will resolve it for me.

In this case, `[Project Page]` 

- is not used more than once
- have two different URLs for iOS and Android.

Hence it doesn't make sense to split them imo.